domains/epicster.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "epicsterrrr",
+    "email": "",
+    "discord": "Epicster#0001"
+  },
+  "record": {
+    "CNAME": "epicsterrrr.github.io"
```

Comment:
this is already pointed to another custom domain. through, you won't be able to setup your is-a.dev domain.